src/main/java/com/startsteps/ecommerceapi/model/Product.java

```diff
@@ -0,0 +1,38 @@
+package com.startsteps.ecommerceapi.model;
+
+import jakarta.persistence.*;
+import jakarta.validation.constraints.NotEmpty;
+import lombok.*;
+import org.hibernate.annotations.CreationTimestamp;
+
+import java.time.LocalDateTime;
+
+@AllArgsConstructor
+@NoArgsConstructor
+@Getter
+@Setter
+@Entity
+@Table(name="Product")
+public class Product {
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    @Column(name="Product_ID")
+    private Long productId;
+    @NonNull
+    @Column(name= "Product_Name")
```

Comment:
either use snake case or camel case. Also you don't need to prefix everything with "Product" since it belongs to the Product table.

Reply:
but don't we need to use snake case convention for sql and camel case for Java? so annotation in Hibernate for naming should correspond to sql naming and java follows camel case convention as usual?